common/lib/xmodule/xmodule/modulestore/split_mongo/caching_descriptor_system.py

```diff
@@ -242,7 +242,7 @@ def xblock_from_json(self, class_, course_key, block_key, block_data, course_ent
         module._edited_on = edit_info.edited_on  # pylint: disable=protected-access
         module.previous_version = edit_info.previous_version
         module.update_version = edit_info.update_version
-        module.source_version = edit_info.source_version
+        module.source_version = edit_info._source_version.version  # pylint: disable=protected-access
```

Comment:
Running this branch locally, I'm running into an error on this line, because `edit_info` doesn't have a `_source_version` attribute. Can we set a default value for this, if it's not set?